### PR TITLE
Update open orders

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: common/exchange.go [hunk 1]
```diff
@@ -41,6 +41,9 @@ type Exchange interface {
 	Address(asset common.Asset) (address ethereum.Address, supported bool)
 	Withdraw(asset common.Asset, amount *big.Int, address ethereum.Address) (string, error)
 	Trade(tradeType string, pair common.TradingPairSymbols, rate, amount float64) (id string, done, remaining float64, finished bool, err error)
+
+	// OpenOrders return open orders from exchange
+	OpenOrders(pair common.TradingPairSymbols) (orders []Order, err error)
```
Comment (secmask) [inline common/exchange.go]: Coinbase exchange need to add this also to satisfy Exchange interface.

File: common/types.go [hunk 2]
```diff
@@ -450,20 +450,21 @@ type AllBalanceResponse struct {
 	Data       map[string]BalanceResponse
 }
 
+// Order accross multiple exchanges
 type Order struct {
-	ID          string // standard id across multiple exchanges
-	Base        string
+	ID          string `json:"ID,omitempty"` // standard id across multiple exchanges
+	Base        string `json:"Base"`
 	Quote       string
```
Comment (secmask) [inline common/types.go]: why some fields does not have json tag?

Comment (halink0803) [reply common/types.go]: this is a legacy declare, it used to return the json like its name (I added json tag only for omit some not important fields). Anw, I will add json tag for all of them.

File: common/exchange.go (hunk 1 above)
Comment (halink0803) [reply common/exchange.go]: Added it.

File: exchange/binance.go [hunk 3]
```diff
@@ -399,6 +399,46 @@ func (bn *Binance) OrderStatus(id string, base, quote string) (string, error) {
...
+		if err != nil {
+			return nil, err
+		}
+		result = append(result, common.Order{
+			OrderID: strconv.FormatUint(order.OrderID, 10),
+			Side:    order.Side,
+			Type:    order.Type,
+			OrigQty: originalQty,
+			Price:   price,
+			Base:    pair.BaseSymbol,
```
Comment (secmask) [inline exchange/binance.go]: please comment detail on the way we fill symbol here.

**Pull request merged**